scenarious/scenario.py

```diff
@@ -36,20 +36,27 @@ def load(cls, source, type_handlers, load_priority=None, reference_handler=None,
         reference_handler = reference_handler or ReferenceHandler()
         entity_store = entity_store or EntityStore()
 
-        return cls(source, type_handlers_by_name, reference_handler=reference_handler, entity_store=entity_store,
+        if not isinstance(sources, list):
```

Comment:
better to check the opposite, because it could be any tuple, list of any other iterable container, right?

Reply:
@mgaitan Since there's already code in to load scenarios as dicts instead of yml strings it's a bit tricky when checking if it's an iterable. So this way I force it to be a list of whatever you pass to `load()`. I'll agree on updating the docstring to make it more explicit.

Reply:
what about `isinstance(sources, (str, dict))`

Reply:
@mgaitan It can also be a file-like object, do you happen to know a proper way to check that?

Reply:
it seems [there is not](https://stackoverflow.com/a/3450959) a canonical way. keep the current code but also accepting tuples and set? `isinstance(... , (list, tuple, set))`